test_maude_data_source is failing with maude 3.2 MAUDE 1.0

Without digging into this, it looks to me like MAUDE is fine but a time-based regression test in the eng_archive test is failing.

```

=================================== FAILURES ===================================
____________________________ test_maude_data_source ____________________________

    @pytest.mark.skipif("not HAS_MAUDE")
    def test_maude_data_source():
        """Fetch data from MAUDE"""
        with fetch.data_source('cxc'):
            datc = fetch.Msid('aogyrct1', date1, date3)
            assert len(datc.vals) == 19
            assert datc.data_source == {'cxc': {'start': '2016:001:00:00:00.287',

                                                'stop': '2016:001:00:00:04.900'}}

    
        with fetch.data_source('maude'):
            datm = fetch.Msid('aogyrct1', date1, date3)
            assert np.all(datm.vals == datc.vals)
            assert not np.all(datm.times == datc.times)
>           assert datm.data_source == {'maude': {'start': '2016:001:00:00:00.203
',
                                                  'stop': '2016:001:00:00:04.815'
,
                                                  'flags': {'tolerance': False, '
subset': False}}}
E           assert {'maude': {'f...0:00:04.900'}} == {'maude': {'fl...0:00:04
.815'}}
E             Detailed information truncated, use "-vv" to show

../../../../../x86_64-linux_CentOS-6/lib/python2.7/site-packages/Ska.engarchive-3.43
.1-py2.7.egg/Ska/engarchive/tests/test_data_source.py:77: AssertionError
=============== 1 failed, 64 passed, 1 skipped in 80.72 seconds =============
```

--- a/Ska/engarchive/tests/test_data_source.py
+++ b/Ska/engarchive/tests/test_data_source.py
@@ -74,8 +74,8 @@ def test_maude_data_source():
         datm = fetch.Msid('aogyrct1', date1, date3)
         assert np.all(datm.vals == datc.vals)
         assert not np.all(datm.times == datc.times)
-        assert datm.data_source == {'maude': {'start': '2016:001:00:00:00.203',
-                                              'stop': '2016:001:00:00:04.815',
+        assert datm.data_source == {'maude': {'start': '2016:001:00:00:00.287',
+                                              'stop': '2016:001:00:00:04.900',
                                               'flags': {'tolerance': False, 'subset': False}}}
 
     with fetch.data_source('cxc', 'maude', 'test-drop-half'):
@@ -83,8 +83,8 @@ def test_maude_data_source():
         assert np.all(datcm.vals == datc.vals)
         assert datcm.data_source == {'cxc': {'start': '2016:001:00:00:00.287',
                                              'stop': '2016:001:00:00:02.337'},
-                                     'maude': {'start': '2016:001:00:00:02.509',
-                                               'stop': '2016:001:00:00:04.815',
+                                     'maude': {'start': '2016:001:00:00:02.593',
+                                               'stop': '2016:001:00:00:04.900',
                                                'flags': {'tolerance': False, 'subset': False}}}
 
 

--- a/Ska/engarchive/version.py
+++ b/Ska/engarchive/version.py
@@ -34,7 +34,7 @@ import os
 ### SET THESE VALUES
 ############################
 # Major, Minor, Bugfix, Dev
-VERSION = (3, 43, 1, False)
+VERSION = (3, 43, 2, False)
 
 
 class SemanticVersion(object):

